### PR TITLE
 Windows 2022 [semver:major] 

### PR DESCRIPTION
**Pull request by mrothstein74**
@felicianotech  @KyleTryon Made requested changes.

### CODE REVIEW

**Comment by EricRibeiro** [comment]
The integration test is failing. 

Also, it might be a good idea to include a test for each image/variant using matrix parameters.

**Comment by mrothstein74** [comment]
> 

@EricRibeiro should be fixed

**Comment by EricRibeiro** [comment]
Awesome! 

Can you remove the [orb.yml](https://github.com/CircleCI-Public/windows-orb/pull/30/files#diff-10f861fbbd55bb7baa29cb4a3b36607a9eda4cf618daa1dd8499d9277a18a54f)? This file shouldn't be pushed to the repo.

Also, add integration tests for the other executors/variants?

**Comment by felicianotech** [comment]
Let's not merge this even if it passes. I want to talk more broadly with the team on this approach to see if we all agree.

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,8 +68,23 @@ workflows:
...
     jobs:
-      - integration-test
-      # Actually publish the orb
+      - test-server-2019:
+          name: test-server-2019-<< matrix.variant >>-<< matrix.version >>
+          matrix:
+            parameters:
+              variant: [vs2019]
+              version: [stable, edge]
+     # - test-server-2019-cuda:
```
Comment (KyleTryon) [inline .circleci/config.yml:77]: Why is this commented out?

Comment (mrothstein74) [reply .circleci/config.yml:77]: > Why is this commented out?

commented out because image name wont exist until vm-service gets updated tomorrow

Comment (mrothstein74) [reply .circleci/config.yml]: @KyleTryon vm-service has now been updated so i was able to put the test back in there.

**Pull request merged**